smslibrary/src/main/java/com/eis/smslibrary/SMSSentBroadcastReceiver.java

```diff
@@ -0,0 +1,78 @@
+package com.eis.smslibrary;
+
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.telephony.SmsManager;
+
+import androidx.annotation.NonNull;
+
+import com.eis.smslibrary.listeners.SMSSentListener;
+
+/**
+ * Broadcast receiver for sent messages, called by Android Library.
+ * Must be instantiated and set as receiver with context.registerReceiver(...)
+ */
+class SMSSentBroadcastReceiver extends BroadcastReceiver {
+
+    private SMSSentListener listener;
+    private SMSMessage message;
+
+    /**
+     * Constructor for the custom {@link BroadcastReceiver}.
+     *
+     * @param message  message that will be sent.
+     * @param listener listener to be called when the operation is completed successfully or not.
```

Comment:
Here there are two repetitions: message message and listener listener..maybe you should write only once these words.

Reply:
Yep you're right, changed it